Tests/ArgumentParserExampleTests/RepeatExampleTests.swift

```diff
@@ -27,6 +27,9 @@ final class RepeatExampleTests: XCTestCase {
   
   func testRepeat_Help() throws {
     let helpText = """
+        OVERVIEW:
+        Utility to repeat an phrase a number of times
```

Comment:
Do we still want this newline here, or should it be collapsed one more?

```
OVERVIEW: Utility to repeat an phrase a number of times
```

Reply:
We absolutely want the newline as discussions are typically multiline. I just needed something to test.

Reply:
Sounds good!